src/views/Routes.tsx

```diff
@@ -0,0 +1,392 @@
+            width: 600,
+            content: <Space direction="vertical" size="small">
+                {routeToAction &&
+                    <>
+                        <Title level={5}>Delete netowork route "{routeToAction ? routeToAction.network_id : ''}"</Title>
+                        <Paragraph>Are you sure you want to delete this route from your account?</Paragraph>
+                    </>
+                }
+            </Space>,
+            okType: 'danger',
+            onOk() {
+                dispatch(routeActions.deleteRoute.request({getAccessTokenSilently:accessToken, payload: routeToAction?.id || ''}));
+            },
+            onCancel() {
+                setRouteToAction(null);
+            },
+        });
+    }
+
+    const filterDataTable = ():Route[] => {
+        const t = textToSearch.toLowerCase().trim()
+        let f:Route[] = filter(routes, (f:Route) =>
+            (f.network_id.toLowerCase().includes(t) ||f.network.toLowerCase().includes(t) || f.description.toLowerCase().includes(t) || t === "")
+        ) as Route[]
+        if (optionAllEnable !== "all") {
+             f = filter(f, (f:Route) => f.enabled)
+        }
+        return f
+    }
+
+    const onClickAddNewRoute = () => {
+        dispatch(routeActions.setSetupNewRouteVisible(true));
+        dispatch(routeActions.setRoute({
+            network: '',
+            network_id: '',
+            description: '',
+            peer: '',
+            masquerade: false,
+            metric: 9999,
+            enabled: true
+        } as Route))
+    }
+
+    const onClickViewRoute = () => {
+        dispatch(routeActions.setSetupNewRouteVisible(true));
+        dispatch(routeActions.setRoute({
+            id: routeToAction?.id || null,
+            network: routeToAction?.network,
+            network_id: routeToAction?.network_id,
+            description: routeToAction?.description,
+            peer: routeToAction?.peer,
+            metric: routeToAction?.metric,
+            masquerade: routeToAction?.masquerade,
+            enabled: routeToAction?.enabled
+        } as Route))
+    }
+
+    const setRouteAndView = (route: RouteDataTable) => {
+        dispatch(routeActions.setSetupNewRouteVisible(true));
+        dispatch(routeActions.setRoute({
+            id: route.id || null,
+            network: route.network,
+            network_id: route.network_id,
+            description: route.description,
+            peer: route.peer,
+            metric: route.metric,
+            masquerade: route.masquerade,
+            enabled: route.enabled
+        } as Route))
+    }
+
+    const showConfirmEnableMasquerade = (record: RouteDataTable, checked: boolean) => {
+        let label = record.network_id ? record.network_id : record.network
+        let tittle = "Enable Masquerade for \"" + label + "\"?"
+        let content = "Enabling this option hide all traffic coming from other NetBird peers through the routing peer address."
```

Comment:
the wording sounds a bit odd.
Maybe this? 
`Enabling this option hides all traffic from other NetBird peers through the routing peer address.`

I also suggest, maybe we put a bit more context for non-experienced users and describe what this option exactly does?